test/testPriceRouter/SequencerPriceRouter.t.sol

```diff
@@ -0,0 +1,102 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.21;
+
+import { SequencerPriceRouter, PriceRouter, Registry, ERC20, IChainlinkAggregator } from "src/modules/price-router/permutations/SequencerPriceRouter.sol";
+import { Math } from "src/utils/Math.sol";
+
+import { Test, stdStorage, StdStorage, stdError, console } from "@forge-std/Test.sol";
+
+// TODO refactor to a multichain test once deploy is merged into branch.
```

Comment:
What do you think about creating an open issue that has a list of reformatting and other TODOs for multichain tests, etc.? That way we can remove this TODO for now.

Reply:
sg2m